homeassistant/auth/providers/command_line.py

```diff
@@ -90,7 +90,7 @@ async def async_validate_login(self, username: str, password: str) -> None:
                     line = _line.decode().lstrip()
                     if line.startswith("#"):
                         continue
-                    key, value = line.split("=", 1)
+                    key, _, value = line.partition("=")
                 except ValueError:
```

Comment:
This won't work because ValueError is intended to catch the partition error.  Better leave as is or handle an empty separator (item 1 in returned tuple) by raising ValueError..